src/core/stream.js

```diff
@@ -123,6 +123,7 @@ var Stream = (function StreamClosure() {
       this.start = this.pos;
     },
     makeSubStream: function Stream_makeSubStream(start, length, dict) {
+      length = length !== undefined ? length : this.bytes.buffer.length;
```

Comment:
Can we please use more "consistent" formatting between the various `makeSubStream`-methods?
I.e. let's change this to the follow, since you also don't need to update the `length` when it's already set.
```suggestion
      if (length === undefined) {
        length = this.bytes.buffer.length;
      }
```

Reply:
Actually, is it even necessary to handle the `length === undefined` case at all in *this* method?
Given that https://github.com/mozilla/pdf.js/blob/ba76dd4000c0efa570558dd56d551b773bb4e856/src/core/stream.js#L36 ought to assign a correct `this.end` value already, since `start + undefined === NaN` and we'd fallback to `this.bytes.length`.